uberfire-rest/uberfire-rest-backend/src/main/java/org/guvnor/rest/backend/cmd/CreateOrgUnitCmd.java

```diff
@@ -41,14 +41,13 @@ public JobResult internalExecute(JobRequest request) throws Exception {
 
         JobResult result = null;
         try {
-            result = helper.createOrganizationalUnit(jobRequest.getJobId(),
-                                                     jobRequest.getSpaceName(),
-                                                     jobRequest.getOwner(),
-                                                     jobRequest.getDefaultGroupId(),
-                                                     jobRequest.getProjects());
+            result = helper.createSpace(jobRequest.getJobId(),
+                                        jobRequest.getSpaceName(),
+                                        jobRequest.getOwner(),
+                                        jobRequest.getDefaultGroupId());
         } finally {
             JobStatus status = result != null ? result.getStatus() : JobStatus.SERVER_ERROR;
-            logger.debug("-----createOrganizationalUnit--- , OrganizationalUnit name: {}, OrganizationalUnit owner: {} [{}]",
+            logger.debug("-----createSpace--- , OrganizationalUnit name: {}, OrganizationalUnit owner: {} [{}]",
```

Comment:
Probably same as above. The log message would be more explanatory if "Space name" and "Space owner" were used.